cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      ""
+    ],
+    "date_published": "2023",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 1,
+      "size": 137529
+    },
+    "experiment": "JADE",
+    "files": [
+      {
+        "checksum": "adler32:46c1eb5f",
+        "size": 137529,
+        "uri": "root://eospublic.cern.ch///eos/opendata/jade/documentation/notes/0Index.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "261300",
+    "title": "JADE Computing Note 0",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "Computing Note"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "Monte Carlo Formats"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      "Nov 09 1978"
+    ],
+    "date_published": "2023",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 1,
+      "size": 306074
+    },
+    "experiment": "JADE",
+    "files": [
+      {
+        "checksum": "adler32:cbb6b90c",
+        "size": 306074,
+        "uri": "root://eospublic.cern.ch///eos/opendata/jade/documentation/notes/10.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "261301",
+    "title": "JADE Computing Note 10",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "Computing Note"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "Vertex chanber software"
+    },
+    "authors": "Hagermann, Kleinworth, Ramcke",
```

Comment:
The "authors" field is a list. If you have three authors, then you should do:

```yaml
    "authors": [
      {
        "affiliation": "Orsay, LAL; Paris, IN2P3; Orsay",
        "name": "Adam-Bourdarios, Claire",
        "orcid": "0000-0002-2634-4958",
        "rorid": "046w3ze59; 03fd77x13; 028rypz17"
      },
      {
        "affiliation": "Royal Holloway, U. of London",
        "name": "Cowan, Glen",
        "orcid": "0000-0001-8363-9827",
        "rorid": "04g2vpn86"
      },
      {
        "affiliation": "INRIA, Saclay; LRI, Paris 11; Orsay",
        "name": "Germain, Cecile",
        "orcid": "0000-0002-9764-9783",
        "rorid": "02kvxyf05; 04e3ktk27; 028rypz17"
      },
]
```

You can obviously leave out the ORCID, RORID, affiliation fields if you don't have them for past authors.

Please note also that we prefer to have full names in the format "Familyname, Firstname".

Reply:
Unfortunately the full names are not given and they would not be easily accessible.

Reply:
BTW in hte note itself I'm seeing initials, perhaps we could at least do "Eckerlin, G" and "Zimmer, M" instead of using just family names?

(We could do this also in later improvements, after merge, if you prefer.)